### PR TITLE
Add road to publish

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+status: publish
+categories: [Writing]
+image:
+image2:
+author: Valentino Urbano
+---
+
+# The Beginning
+
+I'm turning my "Migrating From WordPress To Jekyll" blog post into a small ebook. I will be revisiting each entry and edit it for the medium.
```
Comment (MsSlacker) [inline _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md]: Edit - > editing, keep the tense

File: _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Firstly I had to decide how to publish it. I decided to release it for free online, while at the same time release a pdf and epub version on both Amazon and Gumroad for a small fee (still not decided). I don't expect to get any sale, but it is going to be an interesting process to go through nonetheless both generating and formatting the book as well as figuring out how to set up a page on both Amazon and Gumroad.
```
Comment (MsSlacker) [inline _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md]: Change word order - > online for free

release - > releasing? 

sale - > sales

File: _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+*None of this is set in stone so it might change in the future.*
+
+In the next few days I'll be writing on how to setup a page for a book on any Jekyll based websites.
```
Comment (MsSlacker) [inline _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md]: websites - > website?

File: _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+Writing 200 words a day is great for writing, but I found that I did not have enough time to edit so I am walking a few steps down that both both with [code review fro writing][1] which is open to everyone and with this initiative. I solved the problem on now being able to get enough ideas, but I now have the new problem of feeling like my articles are not edited well due to time concerns. I go more in depth on that topic [in this article][2] so I won't prolong this too much.
```
Comment (MsSlacker) [inline _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md]: REWRITE.

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md (hunk 4 above)
Comment (valeIT) [reply _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md]: Changed quite a bit, also restructured with some titles.

File: _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md [hunk 5]
```diff
@@ -13,18 +13,32 @@ author: Valentino Urbano
...
+## The Concept Behind It
 
-I have been writing this particular series for a while and it is one of me longest articles. I still have a few entries in it to finish what my baseline idea was for it, but I only did really small and rough edits to it while writing and I didn't have the bigger picture of the whole finished piece before so now it can be edited further and improved. I have a lot going on at the moment so I'm not going to assure any specific timing for it, but as with everything you will be able to follow the progress on GitHub.
+Writing 200 words every day is great for generating new ideas and getting into the habit of writing something on a consistent basis.
+Writing so often means that I did not have enough time to edit since I was writing and publishing an article on the same day.
 
+I will be now moving to write one day in advance as explained in the [code review for writing][1] article. The review process is open for everyone to join and comment so feel free to let me know of any mistakes or improvements that could be made.
```
Comment (MsSlacker) [inline _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md]: I will be now changing my schedule to write one day (...)?

File: _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md [hunk 6]
```diff
@@ -13,18 +13,32 @@ author: Valentino Urbano
...
 
-[1]: https://github.com/valeIT/ ...
-[2]: {% post_url Code Review Writing %}
\ No newline at end of file
+## On "Migrating From WordPress To Jekyll"
+
+I have been writing this particular series for a while and it is the longest articles I've written this year.
```
Comment (MsSlacker) [inline _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md]: articles - > article

File: _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md [hunk 7]
```diff
@@ -13,18 +13,32 @@ author: Valentino Urbano
...
+
+I still have a few entries in it to be able to mark it as finished, but I only did small edits to it while writing. I also did not have the bigger picture of how the whole finished piece would look like beforehand[^1]. Now that it is done, it can be edited further and improved.
```
Comment (MsSlacker) [inline _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md]: (...) in it before being able to mark it as finished

File: _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md [hunk 8]
```diff
@@ -13,18 +13,32 @@ author: Valentino Urbano
...
+
+I have a lot going on at the moment so I'm not going to assure any specific timing for it, but as with everything you will be able to follow the progress on GitHub.
```
Comment (MsSlacker) [inline _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md]: assure sounds weird, maybe state?

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md (hunk 5 above)
Comment (valeIT) [reply _posts/2019-10-21-The-Road-To-Publish-My-Short-ebook.md]: Way better

**Review by MsSlacker — APPROVED** [review]

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**